src/contracts/core/AllocationManager.sol

```diff
@@ -53,13 +53,19 @@ contract AllocationManager is
     function slashOperator(
         SlashingParams calldata params
     ) external onlyWhenNotPaused(PAUSED_OPERATOR_SLASHING) {
+        // Assert that the amount to slash is within bounds of 1 and WAD (1e18).
         require(0 < params.wadToSlash && params.wadToSlash <= WAD, InvalidWadToSlash());
 
-        // Check that the operator is registered and slashable
         OperatorSet memory operatorSet = OperatorSet({avs: msg.sender, operatorSetId: params.operatorSetId});
-        bytes32 operatorSetKey = _encodeOperatorSet(operatorSet);
+
+        // Assert that the provided strategies are all whitelisted for the given operator set.
```

Comment:
Not a fan of repeat external calls to avsd, hoping we can merge these contracts.

Reply:
It's possible we can merge the _new_ AVSD functionality (ie. opset functionality) into the ALM. Unsure about merging ALM into AVSD